### PR TITLE
First take on Application Names section

### PR DESCRIPTION
**Pull request by ernesto-jimenez**
Hi @marcoscaceres,

Here's a first take on the section names.

This is not intended to be merged yet but to get your feedback.

It's my first time editing, so it would be great to have your comments.

Cheers,
Ernesto

### CODE REVIEW

File: sections/names.html [hunk 1]
```diff
@@ -1,54 +1,114 @@
...
+  uses whether to use the long or the short name depending on where it's going
+  to be used. E.g.: iOS applications and W3C Widgets</li>
+  <li>Define a default display name and allow fine-grained control to set
+  display names for different use cases. E.g.: Android Manifest file allows
+  fine-grained control on the display name to be shown in the home screen,
+  intents and different activities.</li>
+</ol>
+
+<p>iOS and Android developers also need to define a listings name during the
+application submision process to the store which is used in the store listings.
```
Comment (marcoscaceres) [inline sections/names.html]: typo: submision

File: sections/names.html [hunk 2]
```diff
@@ -1,54 +1,114 @@
...
+  application name is also appended to the window title of the Pinned site
+  application instance.</p>
+  <p>If this element is missing, the document title is used instead.</p>
+</blockquote>
+
+<h4>Safari Mobile</h4>
+
+<p>Since iOS6, apple supports it's own version of "application-name" metadata
+called "apple-mobile-web-app-title". However, it doesn't use "application-name"
+(TODO: VERIFY)</p>
```
Comment (marcoscaceres) [inline sections/names.html]: TODO: we should note the fallback to using the title element.

File: sections/names.html [hunk 3]
```diff
@@ -1,54 +1,114 @@
...
+
+<h4>Chrome Mobile</h4>
+
+<p>TODO: CHECK</p>
```
Comment (marcoscaceres) [inline sections/names.html]: They removed installed web apps support in the latest release :(

File: sections/names.html [hunk 4]
```diff
@@ -1,54 +1,114 @@
...
+<h3>Current tag ussage</h3>
+
+<p>We analysed the <a href="https://gist.github.com/7454160">October data
+set</a> (78,156 sites) to review the ussage of the different meta tags to
+define names:</p>
+
+<ol>
+  <li><strong>"application-name"</strong>: used in 1,571 sites.</li>
+  <li><strong>"apple-mobile-web-app-title"</strong>: used in 244 sites. 50 of
+  those sites used "application-name" too.</li>
```
Comment (marcoscaceres) [inline sections/names.html:84]: Nice find! :)

**Comment by marcoscaceres** [comment]
This is looking good!

**Comment by ernesto-jimenez** [comment]
Fixed the typo and updated the browser status:
- Clarify Safari Mobile falls-back to title and confirmed it works as stated
- Confirmed Chrome Mobile and Firefox for Android only use the title tag

**Comment by marcoscaceres** [comment]
@ernesto-jimenez you ready for me to merge? Or you want me to keep reviewing?

**Comment by ernesto-jimenez** [comment]
I want to double-check the ReSpec output because I think it was doing some weird things with the H4 headers.

Apart from that, I think it should be OK to merge if this looks good enough to you. We have some reviews left on the document, so it'll probably be OK.

**Comment by marcoscaceres** [comment]
Reviewed. All looks good to me!

**Pull request merged**